drivers/metal/metal_objects.mm

```diff
@@ -1093,9 +1047,373 @@
 	}
 }
 
-BoundUniformSet &MDUniformSet::boundUniformSetForShader(MDShader *p_shader, id<MTLDevice> p_device) {
+void MDUniformSet::bind_uniforms_argument_buffers(MDShader *p_shader, MDCommandBuffer::RenderState &p_state) {
```

Comment:
This is the original argument buffers implementation moved to a new method, so we can call the right binding, depending on whether the shader uses argument buffer ABI or classic ABI